build.gradle.kts

```diff
@@ -171,6 +175,23 @@ tasks.register<Task>("copyScripts") {
     dependsOn(copyGradleScripts, copyMavenScripts)
 }
 
+val consumableGradleScripts by configurations.creating {
+    isCanBeConsumed = true
+    isCanBeResolved = false
+    attributes.attribute(Usage.USAGE_ATTRIBUTE, objects.named("gradle-build-validation-scripts"))
```

Comment:
I'm not familiar with the Usage.USAGE_ATTRIBUTE attribute? Can you add a pointer to the review, just for my education, please?

Reply:
[Here are the docs](https://docs.gradle.org/current/userguide/variant_attributes.html#ecosystem_independent_standard_attributes) and [here is the Javadoc](https://docs.gradle.org/current/javadoc/org/gradle/api/attributes/Usage.html)..

Reply:
Thanks.